components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1600,6 +1601,12 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         }
     }
 
+    public async updateProjectSettings(ctx: TraceContext, projectId: string, partialSettings: Partial<ProjectSettings>): Promise<void> {
```

Comment:
As mentioned above: ideally we'd have a single `udpateProjectPartial` to reduce API implementation.